### PR TITLE
node-exporter-adder

### PR DESCRIPTION
**Pull request by Rajat0002**

### CODE REVIEW

File: community_images/node-exporter/ironbank/k8s_coverage.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json params for k8s coverage = $JSON"
+
+NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
+RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
+echo "${RELEASE_NAME}"
+CONTAINER_NAME="${RELEASE_NAME}"
+kubectl logs -n "${NAMESPACE}" "${CONTAINER_NAME}"
```
Comment (Kirandeep-Singh-Khehra) [inline community_images/node-exporter/ironbank/k8s_coverage.sh:21]: @Rajat0002 you can remove `kubectl logs -n ...` as its already present in orchestrator.

File: community_images/node-exporter/bitnami/dc_coverage.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+PROJECT_NAME=$(jq -r '.project_name' < "$JSON_PARAMS")
+
+CONTAINER_NAME="${PROJECT_NAME}"-node-exporter-1
+docker inspect "${CONTAINER_NAME}"
+sleep 5
+
+docker exec "${CONTAINER_NAME}" /opt/bitnami/node-exporter/bin/node_exporter --version
+
+# Check the metrics using cURL for node-exporter
+curl http://localhost:9100/metrics
```
Comment (AnmolVirdi) [inline community_images/node-exporter/bitnami/dc_coverage.sh:26]: Can we use dynamic port mapping here on?
Refer: apache/official

File: community_images/node-exporter/bitnami/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+  node-exporter:
+    image: ${NODE_EXPORTER_IMAGE_REPOSITORY}:${NODE_EXPORTER_IMAGE_TAG}
+    cap_add:
+      - SYS_PTRACE
+    ports:
+      - '9100:9100'
```
Comment (AnmolVirdi) [inline community_images/node-exporter/bitnami/docker-compose.yml:9]: Dynamic port mapping preferred

File: community_images/node-exporter/ironbank/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+  node-exporter:
+    image: ${NODE_EXPORTER_IMAGE_REPOSITORY}:${NODE_EXPORTER_IMAGE_TAG}
+    cap_add:
+      - SYS_PTRACE
+    ports:
+      - '9100:9100'
```
Comment (AnmolVirdi) [inline community_images/node-exporter/ironbank/docker-compose.yml:9]: dynamic port mapping preferred

File: community_images/node-exporter/ironbank/k8s_coverage.sh (hunk 1 above)
Comment (Rajat0002) [reply community_images/node-exporter/ironbank/k8s_coverage.sh:21]: Done

File: community_images/node-exporter/bitnami/dc_coverage.sh (hunk 2 above)
Comment (Rajat0002) [reply community_images/node-exporter/bitnami/dc_coverage.sh]: Updated

**Review by AnmolVirdi — APPROVED** [review]

**Pull request merged**